exec_fd_test.c

```diff
@@ -0,0 +1,13 @@
+#include <common.h>
+#include <exec.h>
+
+int main() {
```

Comment:
Since @mkaim implemented proper kernel parameter passing, could we have a single launcher that passes `run=elfname;arg1;arg2;...` to the kernel?